packages/integration-tests/package.json

```diff
@@ -21,14 +21,18 @@
     "@silverhand/ts-config": "^0.14.0",
     "@types/jest": "^27.4.1",
     "@types/node": "^16.0.0",
+    "core-js": "^3.1.4",
     "eslint": "^8.10.0",
     "got": "^11.8.2",
     "jest": "^27.5.1",
     "node-fetch": "^2.6.7",
     "prettier": "^2.3.2",
+    "react-is": ">=16.8.0",
```

Comment:
why do we need these? didn't any peer deps in `redoc-cli`

https://github.com/Redocly/redocly-cli/blob/master/package.json